pages/[network]/avatar/[address].tsx

```diff
@@ -19,14 +22,15 @@ const CURRENCY_TICKERS = [
     "SOULSTONE_1002",
     "SOULSTONE_1003",
     "SOULSTONE_1004",
-] as const;
+];
```

Comment:
If you didn't want to the array dynamically, it will be better to make the constant array stay in readonly state. (revert)

```suggestion
] as const;
```